regparser/tree/xml_parser/reg_text.py

```diff
@@ -160,8 +161,16 @@ def next_marker(xml_node, remaining_markers):
 def build_from_section(reg_part, section_xml):
     section_texts = []
     nodes = []
+
+    section_no = section_xml.xpath('SECTNO')[0].text
```

Comment:
It's doing the same thing it was doing before, except in a different place;
I didn't write this code, I just moved it up so that the subject text would
be available for stuff I wanted to do.

On Wed, Mar 18, 2015 at 12:19 PM, Shashank Khandelwal <
notifications@github.com> wrote:

> In regparser/tree/xml_parser/reg_text.py
> <https://github.com/cfpb/regulations-parser/pull/264#discussion_r26679083>
> :
>
> > @@ -160,8 +161,16 @@ def next_marker(xml_node, remaining_markers):
> >  def build_from_section(reg_part, section_xml):
> >      section_texts = []
> >      nodes = []
> > +
> > +    section_no = section_xml.xpath('SECTNO')[0].text
>
> What's this doing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/regulations-parser/pull/264/files#r26679083>.
>



-- 
http://www.google.com/profiles/grapesmoker